src/main/java/org/broadinstitute/hellbender/utils/variant/Variant.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.hellbender.utils.variant;
+
+import htsjdk.samtools.util.Locatable;
+
+/**
+ * Variant is (currently) a minimal variant interface needed by the Hellbender pipeline.
+ * This will be expanded as more methods are needed.
+ * NOTE: getStart() and getEnd() are 1-base inclusive (which matches the current GATK tools).
+ * This does not match the GA4GH spec.
+ */
+public interface Variant extends Locatable {
+    String getContig();
+    int getStart();
+    int getEnd();
```

Comment:
Since this interface extends `Locatable`, which already has `getContig()`, `getStart()`, and `getEnd()`, you don't need to repeat their declarations here.